### PR TITLE
Setup plan_node_id for ExchangeOperators

### PR DESCRIPTION
**Pull request by liuyehcf**
Setup plan_node_id for ExchangeOperators and LocalExchangeOperators

### CODE REVIEW

File: be/src/exec/pipeline/operator.h [hunk 1]
```diff
@@ -118,12 +118,19 @@ class Operator {
     // equal to ExecNode::eval_join_runtime_filters, is used to apply bloom-filters to Operators.
     void eval_runtime_bloom_filters(vectorized::Chunk* chunk);
 
+    // 1. (-∞, s_pseudo_plan_node_id_upper_bound] is for operator which is not in the query's plan
+    // for example, LocalExchangeSinkOperator, LocalExchangeSourceOperator
+    // 2. (s_pseudo_plan_node_id_upper_bound, -1] is for operator which is in the query's plan
+    // for example, ResultSink
+    static const int32_t s_pseudo_plan_node_id_for_result_sink;
```
Comment (kangkaisen) [inline be/src/exec/pipeline/operator.h:125]: Why not use the real plan node id?

File: be/src/exec/pipeline/operator.cpp [hunk 2]
```diff
@@ -10,11 +10,18 @@
 #include "util/runtime_profile.h"
 
 namespace starrocks::pipeline {
+
+const int32_t Operator::s_pseudo_plan_node_id_for_result_sink = -99;
```
Comment (kangkaisen) [inline be/src/exec/pipeline/operator.cpp:14]: result_sink should has the real plan node id

File: be/src/exec/pipeline/operator.h (hunk 1 above)
Comment (liuyehcf) [reply be/src/exec/pipeline/operator.h:125]: some operators like `LocalExchangeSourceOperator` which is not in the query plan do not have real plan node id, and these operators are added in addition to adjust the degree of parallelism

File: be/src/exec/pipeline/operator.cpp (hunk 2 above)
Comment (liuyehcf) [reply be/src/exec/pipeline/operator.cpp:14]: Hide ResultSink's plan_node_id

**Review by kangkaisen — APPROVED** [review]

**Review by ZiheLiu — APPROVED** [review]

**Review by dirtysalt — APPROVED** [review]

**Pull request merged**